### PR TITLE
Refactor Gateway Submit

### PR DESCRIPTION
**Pull request by gaoyan1998**
修改提交过程GatewayConfig构建部分，通过json转Modal代替map减少大量配置过程与验证。
Jar作业提交配置目前还未有定论，暂时屏蔽。

### CODE REVIEW

File: dinky-common/src/main/java/org/dinky/utils/SqlUtil.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public static String[] getStatements(String sql, String sqlSeparator) {
             return new String[0];
         }
 
-        String[] splits = sql.replace(";\r\n", ";\n").split(sqlSeparator);
+        String[] splits = sql.replace(";\r\n", ";\n").split(";");
```
Comment (leechor) [inline dinky-common/src/main/java/org/dinky/utils/SqlUtil.java]: should use common sql split character at here? this seems influence 'if (lastStatement.endsWith(SEMICOLON))'

Comment (gaoyan1998) [reply dinky-common/src/main/java/org/dinky/utils/SqlUtil.java]: Sorry, here is another bug, should not be in this pr, I accidentally put the temporary change up, here I will revert

**Review by leechor — COMMENTED** [review]
LGTM

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**